superset/cachekeys/commands/exceptions.py

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from flask_babel import lazy_gettext as _
+
+from superset.commands.exceptions import CommandException
+
+
+class WarmUpCacheParametersExpectedError(CommandException):
+    status = 400
+    message = _(
+        "Malformed request. slice_id or table_name "
+        "and db_name arguments are expected"
+    )
+
+
+class WarmUpCacheChartNotFoundError(CommandException):
+    def __init__(self, chart_id: int):
+        message = f"Chart {chart_id} not found"
+        super().__init__(message)
+
+    status = 404
+
+
+class WarmUpCacheTableNotFoundError(CommandException):
+    def __init__(self, table_name: str, db_name: str):
+        message = f"Table {table_name} wasn't found in the database {db_name}"
```

Comment:
let's make sure we don't log or alert user input

Reply:
fixed